src/main/java/stsjorbsmod/memories/WrathMemory.java

```diff
@@ -86,9 +97,15 @@ private void permanentlyIncreaseCardDamage(AbstractCard card) {
             masterCard.baseDamage += DAMAGE_INCREASE_PER_KILL;
             masterCard.superFlash();
             cardToShowForVfx = masterCard;
+
+            // Because it is the master deck that's saved, we need to track
+            // the card's deviation from base for potential future loads.
+            int count = WrathField.upgradeCount.get(masterCard);
+            WrathField.upgradeCount.set(masterCard, count + 1);
         }
 
         for (AbstractCard instance : GetAllInBattleInstances.get(card.uuid)) {
+            instance.isDamageModified = true;

```

Comment:
I don't think we want to set this; it recolors the number on the card in way that usually implies that the instance is being modified in a non-permanent way. #36 tracks updating the title of the card to note the wrath upgrade; I think that would be a better way to indicate it to the user.

Instead, consider setting the WrathField here on the instances as well; that'll probably prevent some head scratching if someone else ends up picking up #36 :)

Reply:
More subtle interaction with #36: we probably want to patch `AbstractCard.makeStatEquivalentCopy()` to copy over the wrath field (probably our ur/meta numbers, too)

Reply:
Drat, I like to finish the thing before bed, but pushing commits after 1am comes with a certain risk. The isDamageModified shouldn't be there at all.

Thanks for the pointer on `makeStatEquivalentCopy()`.